tmt/package_managers/__init__.py

```diff
@@ -116,7 +116,11 @@ def find_package_manager(name: 'GuestPackageManager') -> 'PackageManagerClass':
 
 def escape_installables(*installables: Installable) -> Iterator[str]:
     for installable in installables:
-        yield shlex.quote(str(installable))
+        if isinstance(installable, (FileSystemPath, PackagePath)):
+            yield str(installable)
+
+        else:
+            yield shlex.quote(str(installable))
```

Comment:
Shouldn't we rather drop this part as well and keep the package specification always escaped? What if user wants to use the `dnf` pattern matching for installing the package? The special characters would be expanded by shell which could bring unexpected result.

Reply:
Isn't it the other way around? This line is there to support the exact use case of dnf's pattern matching; it turns `package: binutils*` into `binutils\*` in `dnf install binutils\*`. opening the door to the unexpected side effects you mentioned.

Reply:
Sorry, I'm not expressing myself precisely enough today. I meant: Shall we drop this change and keep quoting even the `FileSystemPath` so that dnf pattern matching is not expanded there by shell? I thought we agreed that we do not want additional magic for `PackagePath` either and that user has to provide the full path.

Reply:
I see, this I understand. Fixed in 1536a109.

Reply:
Good, thanks! I guess this will affects the tests as well.